### ISSUE TITLE
some bugs

### ISSUE THREAD

**Issue by Marmeladka441** [Dddatt/bss#16]
1. you cannot create a gummy bee
2. closing and opening your save file resets glitter cooldown
3. you can come to a questgiver, claim their quest, go to red cannon and instantly complete them. this was abused alot by some people

**Comment by Dddatt** [comment]
yeah ik but im too lazy to fix them and update and do all that stuff

**Comment by spritestatistics** [comment]
answers to bugs:
1: fixed, however it says it costs 2500 gumdrops, but it actually only costs 1500 (i looked in the code), i discovered this when i formed a gummy bee for less than 2500 (@Dddatt is it meant to be 1500 or 2500?)
2. idk
3. fixed i think
